.github/workflows/auto-docs.yml

```
@@ -0,0 +1,38 @@
+name: Auto Docs
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  generate-docs:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
+
+    - name: Install dependencies
+      run: |
+        python3 -m venv robot-venv
+        source robot-venv/bin/activate
+        pip install -r requirements.txt
+
+    - name: Generate documentation
+      run: |
+        chmod +x scripts/create-docs.sh
+        scripts/create-docs.sh
```

If we push the file attributes once, and do not change it, we do not need to chmod it everytime
```suggestion
      run: cd ${GITHUB_WORKSPACE}/scripts && bash create-docs.sh
```

---

@artur-rs adressed [here](https://github.com/Dasharo/open-source-firmware-validation/pull/352/commits/a2bc617ebb04f86174ec5ae2e46003f1aebd3a4f#diff-ad22275c389220855fb5c424fe7960b59b03afdc63661e3dac08f4bab54c9d45L28-R26)